src/EventService.Data.Provider.MsSql.Ef/Migrations/20230126193800_InitialMigration.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using EventService.Data.Provider.MsSql.Ef;
+using LT.DigitalOffice.EventService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace LT.DigitalOffice.EventService.Data.Provider.MsSql.Ef.Migrations;
+
+[DbContext(typeof(EventServiceDbContext))]
+[Migration("20230126193800_InitialMigration")]
```

Comment:
it's Migration by default, don't need to repeat it in name)
rename to 'InitialTables'